src/scheduler.rs

```diff
@@ -140,6 +155,50 @@ async fn scheduler_loop(
 
         debug!(tick = %tick, "Scheduler ticking");
         schedule_checks(tick);
+        let _: () = connection.set(&progress_key, tick.time().timestamp_nanos_opt().unwrap()).await.expect("Couldn't save progress of scheduler");
```

Comment:
Realised that setting started above means we'll start behind, but never actually catch up since we're always waiting for interval between ticks. Will update to either not overwrite start for now, or to have it perform a quick catch up whenever the current tick to process is < 1 tick behind.